test/performance/tools/common.sh

```diff
@@ -0,0 +1,183 @@
+#!/bin/bash
+
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Setup env vars
+export PROJECT_NAME="knative-gcp-performance"
+export USER_NAME="mako-job@knative-gcp-performance.iam.gserviceaccount.com"
+export TEST_ROOT_PATH="$GOPATH/src/github.com/google/knative-gcp/test/performance"
+export KO_DOCKER_REPO="gcr.io/knative-gcp-performance"
+export TEST_CONFIG_VARIANT="continuous"
+export TEST_NAMESPACE="default"
+export PUBSUB_SERVICE_ACCOUNT="performance-pubsub-test"
```

Comment:
I'm not sure whether it's ok that some variables names are the same in performance tests shell script and e2e tests shell script. For example, in https://github.com/google/knative-gcp/blob/master/test/e2e-common.sh, it also have variable like PUBSUB_SERVICE_ACCOUNT, PUBSUB_SERVICE_ACCOUNT_KEY and PUBSUB_SECRET_NAME